common/persistence/visibility/store/elasticsearch/query_interceptors.go

```diff
@@ -26,8 +26,11 @@ package elasticsearch
 
 import (
 	"strconv"
+	"strings"
 	"time"
 
+	"go.temporal.io/server/common/primitives"
```

Comment:
group `go.temporal.io/server` imports together

Reply:
I originally had it this way, but the CI lint check failed. I ran `goimports` and it reordered the import like so. Is the lint check faulty?